core/src/main/kotlin/no/strooped/view/screen/JoinGameScreen.kt

```diff
@@ -12,9 +12,6 @@ import no.strooped.TextureSizes
 import no.strooped.view.screen.components.TextFieldInput
 import no.strooped.view.screen.components.UIButton
 
-/**
```

Comment:
I think it is good to include this reference some place, so we can remember where we came from, and the also evaulators sees what we have used as inspiration. Maybe place it in the in the README if we do not want to have it in the class documentation

Reply:
I can leave in the code then